### PR TITLE
Refactor/current view message v2

### PR DESCRIPTION
**Pull request by knpausch**
Summary of the work done:

- resubmission of a previous closed PR due to merge conflicts. This version is better and is based off the current main (which includes Jen & Lauren's recent updates)
- This incorporate's Heather's feedback on having some indication of what page we are viewing (all recipes, favorites, etc.). This feature updates the nav bar with a header element with status such as: All Recipes, All Favorite Recipes, and whatever tag or name the user searches by (example: user searches "cookies" and nav bar says "Cookie Recipes" and displays "Oops!" when user enters an invalid entry in the search bar)
- Also incorporated a helper function to capitalize first letters in strings so the title nav bar would look more professional when the use inputs a category or name.

This branch includes:

- header HTML element to store the title
- query selector on new header element
- multiple areas where we update the element's innerText to reflect what view we're on
- formats/capitalizes the user's input (radio button/search bar) so it appears more professional in the nav bar

Contributed to code:

- Kiko

Needs to review:

- Lauren
- Eleanor

Additional notes about any issues/questions remaining around this code:

- Let me know if you like the phrases displayed in the nav bar or if you prefer it to say something better

### CODE REVIEW

File: src/scripts.js [hunk 1]
```diff
@@ -130,6 +134,7 @@ function displayRecipeDetailPage(event) {
     if (user.recipesToCook.length > 0 && user.recipesToCook.includes(foundRecipe)) {
         show(removeRecipeButton)
     } else {hide(removeRecipeButton)}
+    currentView.innerText = foundRecipe.name
```
Comment (Eleanorgruth) [inline src/scripts.js:137]: I don't think that we need the title of the recipe here, I would just do `currentView.innerText = ""`

**Review by FrazierLE — CHANGES_REQUESTED** [review]
Slight bug -- the current message continues to have "favorite" in front of the message even if we're not filtering favorites and we're on the home page

File: src/scripts.js [hunk 2]
```diff
@@ -179,12 +184,14 @@ radioButtons.forEach(button => {
     button.addEventListener('click', () => {
         if(homeView) {
             allRecipes.innerHTML = ''
+            currentView.innerText = capitalizeFirstLetter(button.value) + " Recipes"
```
Comment (Eleanorgruth) [inline src/scripts.js:187]: This isn't working as expected. When on the home page if you click a radio button the inner text says "Favorite (whateverTag) Recipes". We don't want the word favorite here.

**Review by Eleanorgruth — CHANGES_REQUESTED** [review]
See the comment about filtering the home page. 

Also when you filter your favorites view we might want to add the "oops" and show an error, "no favorited recipes meet the search criteria" or something like that?

**Comment by jmyacobn** [comment]
Fully functional, but will submit issue for minor user empathy pieces

**Pull request closed without merging**